Tribler/Core/APIImplementation/LaunchManyCore.py

```diff
@@ -238,19 +238,21 @@ def load_communities():
                     self.dispersy.define_auto_load(PreviewChannelCommunity,
                                                    self.session.dispersy_member, kargs=default_kwargs)
 
-                # Define settings for the Tunnel and Multichain community
-                keypair = self.dispersy.crypto.generate_key(u"curve25519")
-                dispersy_member = self.dispersy.get_member(private_key=self.dispersy.crypto.key_to_bin(keypair),)
+                if self.session.get_tunnel_community_enabled():
 
-                if self.session.get_enable_multichain():
-                    from Tribler.community.multichain.community import MultiChainCommunity
-                    # Start the multichain community and hook in the multichain scheduler.
-                    self.dispersy.define_auto_load(MultiChainCommunity, dispersy_member, load=True)
+                    if self.session.get_enable_multichain():
+                        # If the multichain is enabled, we use the permanent multichain keypair
+                        # for both the multichain and the tunnel community
+                        keypair = self.session.multichain_keypair
+                        dispersy_member = self.dispersy.get_member(private_key=keypair.key_to_bin())
```

Comment:
I think you can immediately say: `private_key=self.session.multichain_keypair.key_to_bin()` so line 246 can be removed.